components/cors-mgt/org.wso2.carbon.identity.cors.mgt.core/src/test/java/org/wso2/carbon/identity/cors/mgt/core/test/CORSManagementServiceTests.java

```diff
@@ -150,17 +149,17 @@ public void testGetCORSOriginsWithApplication() throws CORSManagementServiceExce
         try (Connection connection = IdentityDatabaseUtil.getDBConnection(true)) {
             try {
                 for (String origin : SAMPLE_ORIGIN_LIST_1) {
-                    try (PreparedStatement preparedStatement2 =
-                                 connection.prepareStatement(INSERT_CORS_ORIGIN, RETURN_GENERATED_KEYS)) {
+                    try (NamedPreparedStatement namedPreparedStatement =
+                                 new NamedPreparedStatement(connection, INSERT_CORS_ORIGIN)) {
                         // Origin is not present. Therefore add an origin.
-                        preparedStatement2.setInt(1, SampleTenant.ID);
-                        preparedStatement2.setString(2, origin);
-                        preparedStatement2.setString(3, UUID.randomUUID().toString().replace("-", ""));
-                        preparedStatement2.executeUpdate();
+                        namedPreparedStatement.setInt(1, SampleTenant.ID);
+                        namedPreparedStatement.setString(2, origin);
+                        namedPreparedStatement.setString(3, UUID.randomUUID().toString());
+                        namedPreparedStatement.executeUpdate();
 
                         // Get origin id.
                         int corsOriginId;
-                        try (ResultSet resultSet = preparedStatement2.getGeneratedKeys()) {
+                        try (ResultSet resultSet = namedPreparedStatement.getGeneratedKeys()) {
```

Comment:
Yes because NamedPreparedStatement already handled the expected returning columns through \[1]

[1] https://github.com/wso2/carbon-utils/blob/master/components/org.wso2.carbon.database.utils/src/main/java/org/wso2/carbon/database/utils/jdbc/NamedPreparedStatement.java#L86